packages/lab/src/dialog/DialogHeader.tsx

```diff
@@ -68,12 +68,10 @@ export const DialogTitle = ({
     >
       {status && <StatusIndicator status={status} />}
       <div>
-        {subtitle && (
-          <Text as={"label"} variant="secondary">
-            {subtitle}
-          </Text>
+        {preheader && (
+          <h3 className={withBaseName("preheader")}>{preheader}</h3>
```

Comment:
semantically think this makes sense as a h3 ?

Reply:
A H3 isn't appropriate here, this would make a h3 appear before a h2

Reply:
changed to a div